platform/android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -15,36 +15,55 @@ class Authentication {
 
         fun shouldShowBiometricsPrompt(context: Context): Boolean {
             return biometricsAvailable(context) &&
-                Preferences.getBiometricCheck(context) &&
-                !Preferences.getBiometricEnabled(context) &&
+                Preferences.isBiometricCheck(context) &&
+                !Preferences.isBiometricEnabled(context) &&
                 !Preferences.isFirstRun(context)
         }
 
         fun setUp(
             activity: FragmentActivity,
-            setupComplete: () -> Unit
+            newStatus: Boolean,
```

Comment:
What is the expected behaviour of `setUp` when `newStatus == false`?

Reply:
to disable the setting - I removed the `disable()` I had before and refactored this one for both flows